integration-tests/go.mod

```diff
@@ -22,19 +22,19 @@ require (
 	github.com/onsi/gomega v1.30.0
 	github.com/pelletier/go-toml/v2 v2.2.2
 	github.com/pkg/errors v0.9.1
-	github.com/rs/zerolog v1.31.0
+	github.com/rs/zerolog v1.30.0
 	github.com/scylladb/go-reflectx v1.0.1
 	github.com/segmentio/ksuid v1.0.4
 	github.com/shopspring/decimal v1.3.1
 	github.com/slack-go/slack v0.12.2
 	github.com/smartcontractkit/chainlink-automation v1.0.4
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240620164046-1c03d0a97b20
-	github.com/smartcontractkit/chainlink-testing-framework v1.31.1
+	github.com/smartcontractkit/chainlink-testing-framework v1.30.5
```

Comment:
@shileiwill can you undo this CTF downgrade change

Reply:
This also could be linked - https://github.com/smartcontractkit/chainlink/actions/runs/9609686757/job/26504664318?pr=13644#step:4:770